phangsPipeline/casaStuff.py

```diff
@@ -107,7 +109,11 @@
                            tclean,
                            uvcontsub,
                            visstat)
-    
+
+    # TODO: For now, uvcontsub doesn't work as we want it in newer CASA versions, fall back to old version
+    if casa_version_str >= '6.5.2':
```

Comment:
Does this work?  I'd figure it would be something like:
```
from packaging import version
version.parse(casa_version_str) >= version.parse('6.5.2')
```

Reply:
You're right, pushed this as an update. I've seen it just done in strings before so figured it'd be good, but I guess not